SU2_CFD/src/solvers/CDiscAdjMeshSolver.cpp

```diff
@@ -212,30 +210,28 @@ void CDiscAdjMeshSolver::SetSensitivity(CGeometry *geometry, CSolver **solver, C
   ExtractAdjoint_Variables(geometry, config);
 
   /*--- Store the sensitivities in the flow adjoint container ---*/
-  for (iPoint = 0; iPoint < nPoint; iPoint++) {
+  for (auto iPoint = 0u; iPoint < nPoint; iPoint++) {
```

Comment:
Deleted datatype above is unsigned long, should this maybe be a `0ul` therefore instead of `0u` ? `GetSharpEdge_Distance` also takes an ul. One might end up with uint32_t either way but that is not the case on my machines where ul is uint64_t..but I guess alone 4billion points in one process is probably for the next generation of CPU's :D